skytap/vm_test.go

```diff
@@ -0,0 +1,341 @@
+package skytap
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const (
```

Comment:
need to move this response to `testdata/` fixtures 
https://medium.com/@povilasve/go-advanced-tips-tricks-a872503ac859

Reply:
done